### PR TITLE
455 events homepage

### PR DESCRIPTION
**Pull request by Midnite1212**

### CODE REVIEW

File: ui/src/components/home/EventsSection.js [hunk 1]
```diff
@@ -10,47 +9,112 @@ import {
...
     focusOnSelect: true,
     infinite: false,
     slidesPerRow: 1,
     speed: 500,
     swipeToSlide: true,
     variableWidth: true,
+    slidesToShow: 1,
+    slidesToScroll: 1,
+    nextArrow: <SampleNextArrow />,
+    prevArrow: <SamplePrevArrow />,
```
Comment (bhryck) [inline ui/src/components/home/EventsSection.js:87]: image
please maket the arrow buttons larger.

[nitpick] it should also conditionally not render the left arrow if it's on the first event item and not render the right arrow if its on the last item. but i understand if this is difficult to implement

image

File: ui/src/components/home/EventsSectionCards.js [hunk 2]
```diff
@@ -72,54 +71,39 @@ const EventsSectionCard = (props) => {
...
+              fontSize={['sm', '2xl']}
+              color="#A5CBFF"
+              cursor="pointer"
+              isTruncated
             >
-              Learn more
-            </Button>
-          </HStack>
+              {event.title}
+            </Heading>
```
Comment (bhryck) [inline ui/src/components/home/EventsSectionCards.js:98]: [nitpick] title should be fontweight="semibold", could also add extra top padding or top margin. id suggest making the top and bottom spacing equal to look more like the design in figma.

File: ui/src/components/home/EventsSectionCards.js [hunk 3]
```diff
@@ -72,54 +71,39 @@ const EventsSectionCard = (props) => {
       }
     },
   };
-
   return (
     <>
       <Box borderRadius={10} w={width} h={height} mx={4}>
```
Comment (bhryck) [inline ui/src/components/home/EventsSectionCards.js:76]: [very nitpick] 
image

image

if its not difficult, the card size can be adjusted to be slightly bigger for the mobile version

File: ui/src/components/home/EventsSection.js [hunk 4]
```diff
@@ -70,83 +134,117 @@ const EventsSection = () => {
...
+          )
+        : '',
+    [currentWindow]
+  );
 
   useEffect(() => {
     populateData();
   }, []);
 
+  console.log(computedMargin);
```
Comment (bhryck) [inline ui/src/components/home/EventsSection.js:154]: remove later

File: ui/src/components/home/EventsSection.js [hunk 5]
```diff
@@ -70,83 +134,117 @@ const EventsSection = () => {
...
         whiteSpace="nowrap"
-        marginBottom={["none", "1.5em"]}
+        marginTop="2.5em"
+        marginBottom={['none', '3em']}
         _hover={{
-          overflowX: "auto",
+          overflowX: 'auto',
         }}
       >
         <Slider {...sliderSettings} style={sliderStyle}>
```
Comment (bhryck) [inline ui/src/components/home/EventsSection.js:224]: image
currenty the event cards get cut off as we scroll through the slider. in the current website, as you scroll the cards take up the whole width of the container. 
image

**Review by albertparedandan — APPROVED** [review]

**Pull request merged**